src/config.rs

```diff
@@ -1,11 +1,17 @@
 //! Gex configuration.
 #![allow(clippy::derivable_impls)]
-use std::{fs, path::PathBuf, str::FromStr, sync::OnceLock};
+use std::{collections::HashMap, fs, path::PathBuf, str::FromStr, sync::OnceLock};
 
 use anyhow::{Context, Result};
-use clap::Parser;
-use crossterm::style::Color;
-use serde::Deserialize;
+use clap::{command, Parser};
+use crossterm::{event::KeyCode, style::Color};
+use serde::{
+    de::{self, Visitor},
+    Deserialize,
+};
+
+#[allow(unused_imports)]
+use strum::{EnumIter, IntoEnumIterator};
```

Comment:
If we're only using strum in the unit test, you can move it to `dev-dependencies`, and move this `use` statement into the test module